packages/framework-core/src/services/pub-sub/read-model-pub-sub.ts

```diff
@@ -37,22 +37,22 @@ function filterReadModel(readModel: Record<string, any>, filters?: Record<string
 function filterByOperation(filter: ReadModelPropertyFilter, readModelPropValue: any): boolean {
   for (const [operation, value] of Object.entries(filter as Operation<any>)) {
     switch (operation) {
-      case '=':
+      case 'eq':
         if (readModelPropValue !== value) return false
         break
-      case '!=':
+      case 'ne':
         if (readModelPropValue === value) return false
         break
-      case '<':
+      case 'lt':
         if (readModelPropValue >= value) return false
         break
-      case '>':
+      case 'gt':
         if (readModelPropValue <= value) return false
         break
-      case '>=':
+      case 'gte':
         if (readModelPropValue < value) return false
         break
-      case '<=':
+      case 'lte':
         if (readModelPropValue > value) return false
```

Comment:
Why are the conditions switched here? This looks so weird...

Reply:
It's the way the subscriptions work, they definitely will need some refactoring in the future

Reply:
After a conversation with Adrián, this code is **not** broken